Front-End/index.html

```diff
@@ -2,18 +2,184 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>CONTROL PANEL</title>
+    <title>Admin Panel</title>
     <link rel="stylesheet" href="public/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+<link rel="icon" href="public/images/cloud-icon-white.png" type="image/x-icon"/>
+
 </head>
 <body>
-<h1>CONTROL PANEL</h1>
-<p>The panel for our site.</p>
-<img src="https://prismplus.my/cdn/shop/files/AirconLuna-Web-Thumb3_530x@2x.jpg?v=1714356237" alt="aircon" height="200"
-     class="left-aligned">
-<p class = "red">COMING SOON</p>
-<button class="butonn" type="button">SUBMIT</button>
-<button class="butonn-destroy" type="button">Destroy everything</button>
+    <nav class="navbar">
```

Comment:
Have a clear separation of semantic HTML tags. First, you have the header, then the body and finally the footer. Add the semantic tags and separate accordingly.